entities/entities/fadmin_motd/cl_init.lua

```diff
@@ -12,7 +12,7 @@ local function WorldToScreen(vWorldPos,vPos,vScale,aRot)
 end
 
 function ENT:LoadPage()
-	local Page = GetConVarString("_FAdmin_MOTDPage")
+	local Page = GetConVar("_FAdmin_MOTDPage"):GetString()
```

Comment:
There is much better ways to do this. There is a reason why GetConVarString is deprecated.